assets/scss/_variables_project.scss

```diff
@@ -24,6 +24,10 @@ $navbar-dark-disabled-color: rgba($black, 0.25);
 .lead p {
   font-weight: 400;
   color: $secondary; }
+  
+p {
+  margin-left: 1em;
+  margin-right: 1em; }
```

Comment:
nit: I think that it's standard practice to put the closing bracket on its own line, like in the block below this one. I'm not a CSS expert though.

Reply:
Thanks, this was bugging me as well, so I've tidied up the SCSS in this file. It's looking equivalent to me, but PTAL if you have a couple of minutes.

Reply:
Staged at:

https://5cdb8ad88a8ede00087f96f0--elated-haibt-1b47ff.netlify.com/